packages/web3.js-plugin-fee-estimation/src/plugin.ts

```diff
@@ -0,0 +1,337 @@
+import Web3, {
+  BlockNumberOrTag,
+  BlockTags,
+  Contract,
+  DataFormat,
+  DEFAULT_RETURN_FORMAT,
+  FMT_BYTES,
+  FMT_NUMBER,
+  Numbers,
+  Transaction,
+  Web3PluginBase,
+} from 'web3'
+import { TransactionFactory, TxData } from 'web3-eth-accounts'
+import { estimateGas, formatTransaction } from 'web3-eth'
+import {
+  gasPriceOracleABI,
+  gasPriceOracleAddress,
+} from '@eth-optimism/contracts-ts'
+import { RLP } from '@ethereumjs/rlp'
+
+export default class OptimismFeeEstimationPlugin extends Web3PluginBase {
```

Comment:
```suggestion
export class OptimismFeeEstimationPlugin extends Web3PluginBase {
```
Let's not export as default so later we can export more stuff without it being a breaking change